Evaluator spend project (#3893)
langchain-ai/docs: src/langsmith/evaluators.mdx

Click any evaluator in the table to open its detail view. The detail view has four tabs:

- **Overview**: The evaluator's feedback configuration and prompt or code definition.
- **Traces**: Traces processed by this evaluator across all attached projects and datasets.
- **Projects & Datasets**: The tracing projects and datasets this evaluator is attached to, with each attachment's [weekly spend and limit](/langsmith/evaluator-spend).

## Edit an evaluator

Open an evaluator and update its configuration in the **Overview** tab. Because the evaluator is shared, changes apply across all tracing projects and datasets it is attached to.

## Delete an evaluator

> An evaluator cannot be deleted while it is attached to a tracing project or dataset. To delete an evaluator:

Reviewer: ```suggestion
You cannot delete an evaluator while it is attached to a tracing project or dataset. To delete an evaluator:
```

Author: updated
